references/organization/_03-Vision.md

```diff
@@ -16,11 +16,26 @@ jupyter:
 # <ins>Vision</ins>
 
 
-## At the highest level, how to solve this project’s problem---what will be done in the project? 
+## At the highest level, how to solve this project’s problem---what will be done in the project?
 
-We will demonstrate using simulations how to structurally approach demand modeling and modeling in general while paying attention to the causal structure of how people make their choices, and the data generation process in general. 
-We will build on Brathwaite’s work and review paper and demonstrate with steps how to incorporate and think about causal inference in the context of DCM. We will also demonstrate the value of such an approach by contrasting the outcome of this way of thinking to that of traditional demand modeling where little care is given to the causal structure. 
+We will demonstrate, using simulations and a case study, how to approach travel demand modeling while paying attention to both the causal structure of how people make their choices and the treatment assignment process.
+To do this, we will apply the causal inference workflow outlined in Brathwaite and Walker (2018), and we will augment the workflow with additional details and guidance from our experience's in this project.
+We will also demonstrate the value of such an approach by contrasting the outcome of this way of thinking to that of traditional demand modeling where little care is given to the underlying causal structure of the problem.
```

Comment:
Can we be more specific on how we will demonstrate that in here? Or is it unnecessary?